test/integration/integration_test_fixtures.py

```diff
@@ -46,6 +46,8 @@ def __init__(self, ip_version):
     self.server_ip = "::1" if ip_version == IpVersion.IPV6 else "127.0.0.1"
     self.socket_type = socket.AF_INET6 if ip_version == IpVersion.IPV6 else socket.AF_INET
     self.test_server = None
+    self.test_servers = []
```

Comment:
Looking at `test_server`/`test_servers`, somehow I wonder if we should just rewrite/design this class as if multiple backends is the default case,
And then possibly write a derived convenience class which provides easy access for the single-server case.
Probably out of scope in this PR :-)

Reply:
Agreed, filed #258